admin/securedrop_admin/__init__.py

```diff
@@ -375,11 +375,21 @@ def __init__(self, args: argparse.Namespace) -> None:
                 None,
                 lambda config: True,
             ),
+            (
+                "securedrop_app_pow_on_source_interface",
+                True,
+                bool,
+                "On the Source Interface, enable Tor's proof-of-work defense against "
```

Comment:
Would "Enable Tor's proof-of-work defense against denial-of-service attacks for the Source Interface" be more straight-forward?

Reply:
Yes, this is the correct happy medium between the original and my text.  :-)  Fixed in 40f18b915!